mail/backup_mailer/src/opnsense/mvc/app/models/OPNsense/Backup/MailerSettings.xml

```diff
@@ -0,0 +1,32 @@
+<model>
+    <mount>//system/backup/mailer</mount>
+    <version>1.0.0</version>
+    <description>OPNsense Mailer Backup Settings</description>
+    <items>
+        <MailEnabled type="BooleanField">
+          <default>0</default>
+          <Required>Y</Required>
+        </MailEnabled>
+        <Receiver type="EmailField">
+            <Required>Y</Required>
+        </Receiver>
+        <SmtpHost type="TextField">
+            <Required>Y</Required>
+        </SmtpHost>
+        <SmtpPort type="IntegerField">
+            <Required>Y</Required>
+        </SmtpPort>
+        <SmtpSSL type="BooleanField">
+          <default>1</default>
+          <Required>Y</Required>
+        </SmtpSSL>
+        <SmtpUsername type="TextField"/>
+        <SmtpPassword type="TextField"/>
+        <GpgEmail type="EmailField">
+            <Required>Y</Required>
+        </GpgEmail>
+        <GpgPublicKey type="TextField">
+            <Required>Y</Required>
+        </GpgPublicKey>
```

Comment:
maybe this can be made optional - I had lot's of discussions about enforcing encrypted configs in the nextcloud backup provider. It may be better for yourself if you don't run into the same discussions as I did.